IO/ConsoleUtil.cs

```diff
@@ -1,11 +1,10 @@
 ﻿using System;
+using LocalAdmin.V2.IO.Logging;
 
 namespace LocalAdmin.V2.IO
 {
     public static class ConsoleUtil
     {
-        private static readonly char[] ToTrim = { '\n', '\r' };
```

Comment:
Why did you remove this?

Reply:
To make ^ work, and it was useless anyway, blocking people who actually would want newlines in output.

Reply:
Trim only removed newlines at the beginning and the end

Reply:
I know.